entity-service/src/test/resources/ngsild/authorization/ClientCreateEvent.json

```diff
@@ -0,0 +1,10 @@
+{
+  "operationType": "CREATE",
+  "entityId": "urn:ngsi-ld:Client:191a6f0d-df07-4697-afde-da9d8a91d954",
+  "entityType": "Client",
+  "payload": "{\"id\":\"urn:ngsi-ld:Client:191a6f0d-df07-4697-afde-da9d8a91d954\",\"type\":\"Client\",\"clientId\":{\"type\":\"Property\",\"value\":\"stellio-client\"},\"@context\":[\"https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/authorization/jsonld-contexts/authorization.jsonld\",\"http://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld\"]}",
```

Comment:
I don't get the clientId property

Reply:
It is the client_id, the one we use when configuring a client. Not used as part of the authorization process, but handy to have in the graph if you have to search for a client. Similar to the username of an User actually.